ACE/ace/Shared_Memory_Pool.cpp

```diff
@@ -123,7 +123,7 @@ ACE_Shared_Memory_Pool::commit_backing_store_name (size_t rounded_bytes,
                       "exceeded max number of segments = %d, base = %u, offset = %u\n",
                        counter,
                        this->base_addr_,
-                       offset),
+                       static_cast<unsigned int>(offset)),
                       -1);
```

Comment:
The variable is of type off_t, I cannot find the corresponding format specifier macro in ACE for the type. It may be equivalent to ssize_t. However, I couldn't find any documentation to guarantee that. At least, using cast won't break any code on the platform I haven't tried yet.